automation/src/test/scala/org/broadinstitute/dsde/test/api/RawlsApiSpec.scala

```diff
@@ -457,114 +466,117 @@ class RawlsApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLike w
-//
-//          // may need to wait for Cromwell to start processing workflows.  just take the first one we see.
-//
-//          val submissionPatience = PatienceConfig(timeout = scaled(Span(5, Minutes)), interval = scaled(Span(20, Seconds)))
-//          implicit val patienceConfig: PatienceConfig = submissionPatience
-//
-//          val firstWorkflowId = eventually {
-//            val (status, workflows) = Rawls.submissions.getSubmissionStatus(projectName, workspaceName, submissionId)
-//
-//            withClue(s"Submission $projectName/$workspaceName/$submissionId: ") {
-//              workflows should not be (empty)
-//              workflows.head
-//            }
-//          }
-//
-//          // retrieve the workflow's metadata.
-//          // Orchestration times out in 1 minute, so we want to be well below that
-//
-//          // we also need to check that it returns *at all* in under a minute
-//          // `eventually` won't cover this if the call itself is slow and synchronous
-//
-//          val myTimeout = Timeout(scaled(Span(45, Seconds)))
-//          val myInterval = Interval(scaled(Span(10, Seconds)))
-//
-//          implicit val ec: ExecutionContextExecutor = system.dispatcher
-//
-//          def cromwellMetadata(wfId: String) = Future {
-//            Rawls.submissions.getWorkflowMetadata(projectName, workspaceName, submissionId, wfId)
-//          }.futureValue(timeout = myTimeout)
-//
-//          val subworkflowIds = eventually(myTimeout, myInterval) {
-//            val subIds = parseSubWorkflowIdsFromMetadata(cromwellMetadata(firstWorkflowId))
-//            withClue(s"Workflow $projectName/$workspaceName/$submissionId/$firstWorkflowId: ") {
-//              subIds.size shouldBe scatterWidth
-//            }
-//            subIds
-//          }
-//
-//          // can we also quickly retrieve metadata for a few of the subworkflows?
-//
-//          Random.shuffle(subworkflowIds.take(10)).foreach {
-//            cromwellMetadata(_)
-//          }
-//
-//          // clean up: Abort and wait for one minute or Aborted, whichever comes first
-//          // Timeout is OK here: just make a best effort
-//
-//          Rawls.submissions.abortSubmission(projectName, workspaceName, submissionId)
-//
-//          val abortOrGiveUp = retryUntilSuccessOrTimeout()(timeout = 1.minute, interval = 10.seconds) { () =>
-//            Rawls.submissions.getSubmissionStatus(projectName, workspaceName, submissionId) match {
-//              case (status, _) if status == "Aborted" => Future.successful(())
-//              case (status, _) => Future.failed(new Exception(s"Expected Aborted, saw $status"))
-//            }
-//          }
-//
-//          // wait on the future's execution
-//          abortOrGiveUp.futureValue
-//        }
-//      }
-//
-//    }
+    "should retrieve metadata with widely scattered sub-workflows in a short time" ignore {
```

Comment:
This is all commented out on the left. Is the ignore the difference here? And/Or is it functionally equivalent?

Reply:
I believe the functionality is equivalent. This is what I've done:
- I've uncommented it and put in an ignore because this way it'll still compile and be kept up-to-date with the product code.
- I've made it use the new `withTemporaryBillingProject` fixture
- The submissions api has changes since this was commented out so this was never updated (hence i'm switching to `ignore` to enforce maintenance). I've added the following arguments to make it compile: 
```scala
            deleteIntermediateOutputFiles = true,
            useReferenceDisks = false,
            memoryRetryMultiplier = 1.2
```
I'm not sure if these are correct. I don't think it's our job to make this work honestly as it was commented out.